paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentMethodsUiExtension.kt

```diff
@@ -23,12 +23,20 @@ internal fun PaymentMethod.getSavedPaymentMethodIcon(
             )
         }
         PaymentMethod.Type.SepaDebit -> getSepaIcon(showNightIcon = showNightIcon)
-        PaymentMethod.Type.USBankAccount -> usBankAccount?.bankName?.let { TransformToBankIcon(it) }
+        PaymentMethod.Type.USBankAccount -> TransformToBankIcon(usBankAccount?.bankName)
```

Comment:
Better to fall back to the default bank icon than to `stripe_ic_paymentsheet_card_unknown_ref`.